src/components/AttachmentPicker/launchCamera/launchCamera.ios.ts

```diff
@@ -1,24 +1,21 @@
 import {launchCamera} from 'react-native-image-picker';
 import {PERMISSIONS, request, RESULTS} from 'react-native-permissions';
+import type {Callback, CameraOptions} from './types';
+import {ErrorLaunchCamera} from './types';
 
 /**
  * Launching the camera for iOS involves checking for permissions
  * And only then starting the camera
  * If the user deny permission the callback will be called with an error response
  * in the same format as the error returned by react-native-image-picker
- * @param {CameraOptions} options
- * @param {function} callback - callback called with the result
  */
-export default function launchCameraIOS(options, callback) {
+export default function launchCameraIOS(options: CameraOptions, callback: Callback) {
```

Comment:
For all launchCamera implementations we need to have one common type that is used for these functions, you can read more about this in TS guidelines. `src/libs/ComposerUtils/getNumberOfLines` is a good example how to do it